docker-compose.yml

```
@@ -1,9 +1,36 @@
 version: '2'
 services:
+  fleet:
+    build:
+      context: .
+      dockerfile: Dockerfile.compose
+    image: kolide-local
+    command:
+      - /bin/bash
+      - "-c"
+      - "(
+          while ! grep -q 00:01BB /proc/net/tcp6;do echo wait;sleep 1;done; \
+          fleetctl config set --address https://localhost:443 --tls-skip-verify; \
+          fleetctl setup --email kolide@kolide.com --password kolide; \
```

If this is an example, it should be in `example.com` not kolide.com
```suggestion
          fleetctl setup --email kolide@example.com --password kolide; \
```